auxx/Main.hs

```diff
@@ -4,38 +4,59 @@ module Main
 
 import           Universum
 
-import           Control.Exception.Safe (handle)
-import           Data.Maybe (fromMaybe)
-import           Formatting (sformat, shown, (%))
-import           Mockable (Production (..), runProduction)
-import qualified Network.Transport.TCP as TCP (TCPAddr (..))
+import           Control.Exception.Safe
+    (handle)
+import           Data.Maybe
+    (fromMaybe)
+import           Formatting
+    (sformat, shown, (%))
+import           Mockable
+    (Production (..), runProduction)
+import qualified Network.Transport.TCP as TCP
+    (TCPAddr (..))
 import qualified System.IO.Temp as Temp
-import           System.Wlog (LoggerName, logInfo)
+import           System.Wlog
+    (LoggerName, logInfo)
```

Comment:
That whole line would have fit in 80 columns! Why is it being wrapped?

Reply:
:man_shrugging: